include/gpac/constants.h

```diff
@@ -631,6 +631,9 @@ typedef enum
 
 	GF_CODECID_ALAC = GF_4CC('A','L','A','C'),
 
+	/*! codecid for DTS:X Profile 2 audio */
+	GF_CODECID_DTSX = GF_4CC('d','t','s','x'),
```

Comment:
It already existed under the name ```GF_CODECID_DTS_X```.

Reply:
I will correct that and upload the fixed version soon.